organization/annual-reports/annual-report-2021.md

```diff
@@ -0,0 +1,216 @@
+
+In October, we started working with Matthew Claudel to explore open source software opportunities in North America.
+
+### Codebase stewardship
+
+In 2021, we had 2 codebases in Incubation (Signalen and OpenZaak) and one in Assessment (Omgevingsbeleid). We also worked closely with Demodam.
+
+#### Signalen
+
+* In February, the [Signalen codebase started incubation](https://blog.publiccode.net/news/2021/02/04/signalen-is-in-incubation.html) with the Foundation for Public Code.
+* Signalen had  its first replicators in 2021. Starting with 's-Hertogenbosch , there were 3 by the end of the year.
+* [Signalen hosted their 2nd community day](https://signalen.org/news/2021-03-29-verslag-community-meetup/) in March, attracting over 55 potential replicators and vendors.
+* There’s increasing international interest in Signalen - Ghent (Belgium) joined the community day, and Helsingborg (Sweden) tested a demo version.
+* Signalen moved closer to Standard for Public Code compliance by fully meeting the criteria [Publish with an open license](https://standard.publiccode.net/criteria/open-licenses.html).
+
+#### Omgevingsbeleid
+
+* The codebase stewards and Omgevingsbeleid team designed and ran workshops on essential skills for open technical and product management of a public codebase.
+* Improving towards meeting more of the Standard for Public Code criteria, the Omgevingsbeleid developers began creating automated tests suites for the codebases and measuring code coverage. 
+
+#### OpenZaak
+
+* The OpenZaak community consolidated healthy community practices. They established a process for vulnerability reporting and set up a mailing list for upcoming early release notices.
+* [OpenZaak moved closer to Standard for Public Code compliance](https://github.com/open-zaak/open-zaak/blob/main/docs/introduction/open-source/public_code.md) by fully meeting the criteria Make contributing easy, Use open standards and Document codebase maturity.
+
+Over the course of the year, the codebase stewards also met with additional codebases to explore codebase stewardship.
+
+### Ecosystem
+
+In 2021, we were an active member of our broader ecosystem, contributing our time and resources to strengthen our community.
+
+In June, we were proud to be a founding member of the [OSPO Alliance](https://ospo.zone/about/), a new coalition to help organizations discover and understand open source.
+
+We launched a  free [online course introducing the Standard for Public Code](https://citybycity.academy/course/standards-for-smart-cities), hosted by Open and Agile Smart Cities' CITYxCITY online learning platform.
+
+In 2021, we spoke at 18 events and conferences for our broader community. We mentored at and helped organize 2 hackathons.
+
+We held 20 community calls in 2021, covering the Standard for Public Code and the Foundation for Public Code’s work. We also started pruning the backlog of issues for the Standard for Public Code in the open in the spring (April).
```

Comment:
I just spotted that we on line 82 say that backlog pruning started in May and here in April. We had our first one as a regular community call in April and then as a dedicated backlog pruning alongside the community calls from May. Not sure how we want to record that or if it is important. Perhaps it is easiest to go with April on both.

Reply:
I've gone with May on both, because the emphasis is on developing the practice, not doing it as a one-off.